pkg/cmd/render/render.go

```diff
@@ -62,6 +66,11 @@ func newRenderCommand(testOverrides ...func(*renderOpts)) *cobra.Command {
 		lockHostPath:   "/var/run/kubernetes/lock",
 		etcdServerURLs: []string{"https://127.0.0.1:2379"},
 		etcdServingCA:  "root-ca.crt",
+
+		// by default, we are giving the load balancer 20s to remove the
+		// bootstrap kube-apiserver from its pool after TERM signal is
+		// sent to the kube-apiserver on the bootstrap node.
+		shutdownDelayDuration: 20 * time.Second,
```

Comment:
What is the default value for `shutdownDelayDuration` ?
It looks like it is `0`, does it mean that the bootstrap api server didn't give any time to LBs to react ?

Reply:
iiuc its ) for SNO and 129 for AWS:
https://github.com/openshift/cluster-kube-apiserver-operator/blob/37df1b1f80d3be6036b9e31975ac42fcb21b6447/pkg/operator/configobservation/apiserver/observe_termination_duration.go#L37-L46